client/src/components/forms/moodForm/InputValues.js

```diff
@@ -281,35 +289,43 @@ const InputValues = () => {
     );
   };
 
-  const activitySelect = activity => {
-    const sleepActivity = (activity) => {
+  const activitySelect = moods => {
+    const sleepActivity = moods => {
       if (activity === sleep) {
-        return <>{sleepInput()}</>;
+        
       } else {
+        
       }
     };
 
     return (
-      <FormControl className={classes.formControl}>
-        <InputLabel id="activity-select-label">Choose Activity</InputLabel>
-        <Select
-          labelId="activity-select-label"
-          id="activity-select"
-          value={activity}
-          onChange={handleActivityChange}
-        >
-          <MenuItem value={sleep}>Sleep</MenuItem>
-          <MenuItem value={meditation}>Meditation</MenuItem>
-          <MenuItem value={exercise}>Excerise</MenuItem>
-          <MenuItem value={medication}>Medication</MenuItem>
-          <MenuItem value={yoga}>Yoga</MenuItem>
-        </Select>
-        {sleepActivity(activity)}
-      </FormControl>
+      <FormGroup>
+        <FormControl className={classes.formControl}>
+          <InputLabel id="activity-select-label">Choose Activity</InputLabel>
+          <Select
+            labelId="activity-select-label"
+            id="activity-select"
+            value={activity}
+            onChange={handleActivityChange}
+          >
+            <MenuItem value={sleep}>Sleep</MenuItem>
+            <MenuItem value={meditation}>Meditation</MenuItem>
+            <MenuItem value={exercise}>Excerise</MenuItem>
+            <MenuItem value={medication}>Medication</MenuItem>
+            <MenuItem value={yoga}>Yoga</MenuItem>
```

Comment:
These will not work, you are setting the value to the value of what moodStates has, which is null (the default value you set them to). That is why it appears that all these are selected. you want these values to be a unique identifier for each type of activity allowed and set that to `activity`, you then use `activity` to determine what to display next